composer.json

```diff
@@ -14,7 +14,7 @@
   "prefer-stable": true,
     "require": {
         "php": ">=7.4",
-        "symfony/symfony": "^4.4.30",
+        "symfony/symfony": "^4.4.37",

```

Comment:
@EstelleMyddleware , we should avoid to upgrade Symfony on a hotfix version if it is possible. 
If we have to upgrade Symfony, we should request non regression tests from @RomanFaure .

Reply:
@Myddleware Normally it should be safe to upgrade as it is only a minor version https://symfony.com/doc/current/contributing/community/releases.html#backward-compatibility